metricbeat/magefile.go

```diff
@@ -181,3 +171,18 @@ func GoIntegTest(ctx context.Context) error {
 	mg.Deps(Fields)
 	return devtools.GoTestIntegrationForModule(ctx)
 }
+
+// PythonIntegTest executes the python system tests in the integration
+// environment (Docker).
+// Use NOSE_TESTMATCH=pattern to only run tests matching the specified pattern.
+// Use any other NOSE_* environment variable to influence the behavior of
+// nosetests.
```

Comment:
Nice. It'd be nice to support `MODULE` too, but it can be a different change.

Reply:
Is `MODULE` already honored on the Python side and just needs passed-through? I see it in the Go tests. Or did you mean to implement support for honoring `MODULE`?

If it's just a pass-through I'll do that now.

Reply:
I think it is already passed through and more things will be needed, let's do it in a future change.